packages/dhis2/src/Adaptor.js

```diff
@@ -781,6 +907,29 @@ export function dv(dataElement, value) {
   return { dataElement, value };
 }
 
+export function shouldUseNewTracker(resourceType) {
+  return /^(trackedEntityInstances|tracker\/(enrollments|relationships|events|trackedEntities)|enrollments|relationships|events|trackedEntities)$/.test(
```

Comment:
This regex looks over complicated! 

I think this is all you need:
```
return /(tracker\/|enrollments|relationships|events|trackedEntities)/.test(resourceType)
```

Now anything with `tracker/` in the resource type will use the new tracker, plus the 4 specific resources we're looking for.